jetbrains-core/src/software/aws/toolkits/jetbrains/core/gettingstarted/editor/GettingStartedPanel.kt

```diff
@@ -3,17 +3,338 @@
+                        }
+                    }
+                }
+            )
+
+            border = IdeBorderFactory.createRoundedBorder().apply {
+                setColor(PanelConstants.TEXT_FONTCOLOR)
+                preferredSize = Dimension(PanelConstants.PANEL_WIDTH, PanelConstants.PANEL_HEIGHT)
+            }
+        }
+    }
+
+    class PanelAuthBullets(private val panelTitle: String) : BorderLayoutPanel() {
+
+        val codeWhispererBulletsLists: List<BulletAuthPanel> = listOf(
+            BulletAuthPanel(PanelConstants.COMMIT_ICON, message("iam_identity_center.name"), message("aws.onboarding.getstarted.panel.idc_row_comment_text")),
+            BulletAuthPanel(PanelConstants.COMMIT_ICON, message("aws_builder_id.service_name"), "Need to insert tagline"),
+            BulletAuthPanel(PanelConstants.CANCEL_ICON, message("settings.credentials.iam"), message("aws.getstarted.auth.panel.notSupport_text"), false)
+        )
+
+        val resourceBulletsLists: List<BulletAuthPanel> = listOf(
+            BulletAuthPanel(PanelConstants.COMMIT_ICON, message("iam_identity_center.name"), message("aws.onboarding.getstarted.panel.idc_row_comment_text")),
+            BulletAuthPanel(PanelConstants.CANCEL_ICON, message("aws_builder_id.service_name"), "Need to insert tagline", false),
+            BulletAuthPanel(PanelConstants.COMMIT_ICON, message("settings.credentials.iam"), message("aws.getstarted.auth.panel.notSupport_text"))
+        )
+
+        val codeCatalystBulletsLists: List<BulletAuthPanel> = listOf(
+            BulletAuthPanel(PanelConstants.CANCEL_ICON, message("iam_identity_center.name"), message("aws.getstarted.auth.panel.notSupport_text"), false),
+            BulletAuthPanel(PanelConstants.COMMIT_ICON, message("aws_builder_id.service_name"), "Need to insert tagline"),
+            BulletAuthPanel(PanelConstants.CANCEL_ICON, message("settings.credentials.iam"), message("aws.getstarted.auth.panel.notSupport_text"), false)
+        )
+
+        init {
+
+            val serviceTitleMap: MutableMap<String, List<BulletAuthPanel>> = HashMap()
+            serviceTitleMap.put(message("codewhisperer.experiment"), codeWhispererBulletsLists)
+            serviceTitleMap.put(message("aws.getstarted.resource.panel_title"), resourceBulletsLists)
+            serviceTitleMap.put(message("caws.devtoolPanel.title"), codeCatalystBulletsLists)
+
+            addToCenter(
+                panel {
+                    indent {
+
+                        row {
+                            panelTitle.let { label(it).bold().applyToComponent { foreground = PanelConstants.TITLE_TEXT_FONTCOLOR } }
+                        }
+
+                        val defaultList: List<BulletAuthPanel> = mutableListOf()
+
+                        val bulletsList: List<BulletAuthPanel> = serviceTitleMap[panelTitle] ?: defaultList
+
+                        panelTitle.let {
```

Comment:
why does this need to be in a `let{ }` block?

Reply:
It doesn't required, fixed it